sim/core/target.go

```diff
@@ -225,6 +225,8 @@ func GetWeaponSkill(unit *Unit, weapon *Item) float64 {
 		default:
 			return 0
 		}
+	} else if weapon.RangedWeaponType == proto.RangedWeaponType_RangedWeaponTypeUnknown {
```

Comment:
I think this will completely prevent the one-handed weapon skill logic from triggering below because the above condition is the same but `!=`. We probably need to check that the hand type is HandTypeUnknown and that the RangedWeaponType is also unknown for this specific scenario. Could you also leave a comment clarifying that it's used specifically for Paladin's Hammer of Wrath since that's the only case we know about?

Reply:
fixed